src/components/modals/ant-management/ReassignNameModal/ReassignNameModal.tsx

```diff
@@ -0,0 +1,643 @@
+
+  function handleClose() {
+    setWorkflow(undefined);
+    setNewAntProcessId('');
+    setShow(false);
+  }
+
+  async function handleReassign() {
+    try {
+      if (!wallet?.contractSigner) {
+        throw new Error('No ArConnect Signer found');
+      }
+      if (!walletAddress)
+        throw new Error('Must connect to Reassign the domain');
+
+      const previousState: SpawnANTState = {
+        controllers: domainData.controllers,
+        records: domainData.records,
+        owner: walletAddress.toString(),
+        ticker: domainData.info.Ticker,
+        name: domainData.info.Name,
+        description: domainData.info.Description,
+        keywords: domainData.info.Keywords,
+        balances: domainData.state.Balances,
+      };
+
+      const result = await dispatchANTInteraction({
+        signer: wallet.contractSigner,
+        payload: {
+          name,
+          arioProcessId,
+          // if we provide newAntProcessId it will skip spawning the ant and ignore previousState
+          newAntProcessId:
+            antType === REASSIGN_NAME_WORKFLOWS.EXISTING
+              ? newAntProcessId
+              : undefined,
+          previousState:
+            antType === REASSIGN_NAME_WORKFLOWS.NEW_BLANK
+              ? undefined
+              : previousState,
+          luaCodeTxId: DEFAULT_ANT_LUA_ID,
+        },
+        processId,
+        workflowName: ANT_INTERACTION_TYPES.REASSIGN_NAME,
+        dispatch: dispatchTransactionState,
+        owner: walletAddress.toString(),
+        ao: aoClient,
+      });
+      eventEmitter.emit('success', {
+        message: (
+          <div>
+            <span>
+              Reassign Name completed with transaction ID:{' '}
+              <ArweaveID
+                characterCount={8}
+                shouldLink={true}
+                type={ArweaveIdTypes.INTERACTION}
+                id={new ArweaveTransactionID(result.id)}
+              />
+            </span>
+          </div>
+        ),
+        name: ANT_INTERACTION_TYPES.REASSIGN_NAME,
+      });
+      queryClient.resetQueries({
+        queryKey: ['handlers', processId],
+      });
+      queryClient.resetQueries({
+        queryKey: ['domainInfo', name],
+      });
+      queryClient.resetQueries({
+        queryKey: ['domainInfo', processId],
+      });
+
+      dispatchArNSUpdate({
+        walletAddress: walletAddress,
+        arioProcessId,
+        dispatch: dispatchArNSState,
+        emitter: arnsEmitter,
+        ao: aoClient,
+      });
+      handleClose();
+    } catch (error) {
+      eventEmitter.emit('error', error);
+    }
+  }
+
+  if (!show) return <></>;
+
+  if (!workflow) {
+    return (
+      <div className="modal-container">
+        <div className="flex flex-col rounded-md border border-dark-grey bg-foreground p-6 w-[32rem] gap-8 text-white">
+          <div className="flex flex-row justify-between items-center">
+            <h2 className="text-[24px]">Name Reassignment</h2>
```

Comment:
does `test-[24px]` become responsive by override elsewhere? I would expect a callout to a text variable that handles different screen widths

Reply:
especially since they're hard coded throughout. getting each view right for each screen widths would prove more challenging without some shared responsiveness in the app

Reply:
not necessary to do any of this in this PR's context, just calling out some nits

Reply:
Good point, i should be using rem here